### PR TITLE
Mg query log folder fix

### PR DESCRIPTION
**Pull request by Josipmrden**
`--query-log-directory` set to be empty at the beginning so it is not generated inside the data directory. Also set query log directory to be a runtime flag so it can be set easily.

### CODE REVIEW

**Comment by Josipmrden** [comment]
### Standard development
- [ ] Update unit/E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch this branch


### CI Testing Labels
- [ ] Select the appropriate CI test labels _(CI -build=**build-name** -test=**test-suite**)_


### Documentation checklist
- [ ] Add the documentation label
- [ ] Add the bug / feature label
- [ ] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [ ] Write a release note, including added/changed clauses
    - **[ Release note text ]**
- [ ] **[ Documentation PR link memgraph/documentation#XXXX ]**
    - [ ] Is back linked to this development PR
- [ ] **[ Tag someone from docs team ]**

File: src/flags/query.cpp [hunk 1]
```diff
@@ -15,4 +15,4 @@
 
 // Logging flags
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-DEFINE_string(query_log_directory, "mg_query_log", "Path to directory where the query logs should be stored.");
+// DEFINE_string(query_log_directory, "", "Path to directory where the query logs should be stored.");
```
Comment (gitbuda) [inline src/flags/query.cpp:18]: code commented out -> remove

File: src/flags/query.hpp [hunk 2]
```diff
@@ -15,4 +15,4 @@
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 // DECLARE_bool(cartesian_product_enabled);  Moved to run_time_configurable
 
-DECLARE_string(query_log_directory);
+// DECLARE_string(query_log_directory);
```
Comment (gitbuda) [inline src/flags/query.hpp:18]: code commented out -> remove

**Comment by gitbuda** [comment]
But how is this solving a problem, because by package default the session trace log folder should be somewhere under `/var/log/memgraph`?

**Comment by Josipmrden** [comment]
> But how is this solving a problem, because by package default the session trace log folder should be somewhere under `/var/log/memgraph`?

Nothing in Memgraph is by default under /var/log. Not even the --log-file flag

**Comment by gitbuda** [comment]
I think we should add the "package default" value under https://github.com/memgraph/memgraph/blob/master/config/flags.yaml

**Review by gitbuda — APPROVED** [review]

**Pull request merged**